quantum-adapters/kiwoom-adapter/src/kiwoom_api/external/naver_news_client.py

```
@@ -0,0 +1,296 @@
+            
+            async with httpx.AsyncClient(timeout=self.timeout) as client:
+                response = await client.get(
+                    self.base_url,
+                    params=params,
+                    headers=self.headers
+                )
+                
+                if response.status_code == 200:
+                    return response.json()
+                elif response.status_code == 403:
+                    print(f"❌ 네이버 API 권한 오류 (403): API 설정을 확인하세요")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)